src/interpret.jl

```diff
@@ -7,7 +7,7 @@ function sbmlPiecewise(args...)
     elseif length(args) >= 3
         IfElse.ifelse(args[2], args[1], sbmlPiecewise(args[3:end]...))
     else
-        throw(DomainError(args, "malformed piecewise SBML function"))
+        IfElse.ifelse(args[2], args[1], NaN)
```

Comment:
this could work but we need an user-supplied "bottom" value instead of the NaN here.

Does IfElse have anything like that?

Reply:
Not that I am aware of.